app/static/js/initialize_map.js

```diff
@@ -0,0 +1,285 @@
+geocoder = new google.maps.Geocoder();
+address = document.getElementById("address");
+
+//Global array for map markers
```

Comment:
Nit: Commetns will be easier to read if you have spaces between `//` and comment text like

```javascript
// Global array for map markers
```